src/Events/Admin/Settings/Filter_Bar_Upsell.php

```diff
@@ -9,10 +9,10 @@
 
 namespace TEC\Events\Admin\Settings;
 
-use Tribe\Events\Admin\Settings;
-use Tribe__Settings_Tab;
 use TEC\Common\Contracts\Service_Provider;
+use Tribe__Settings_Tab;
 use Tribe__Template;
+use Tribe\Events\Admin\Settings;
```

Comment:
While we're here, could we also remove the 2 extra lines between this and the start of the class docblock?